openapi.yaml

```diff
@@ -599,6 +599,273 @@ paths:
+      - $ref: '#/components/parameters/pageSize'
+      tags:
+      - Account
+      summary: Entity Transfers List
+      description: >
+        Returns a collection of all created and accepted Entity Transfers for this account, regardless of the user
+        that created or accepted the transfer.
+
+
+        This command can only be accessed by the unrestricted users of an account.
+      operationId: getEntityTransfers
+      x-linode-cli-action: list
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - account:read_only
+      responses:
+        '200':
+          description: >
+            Returns a paginated list of Entity Transfer objects containing the details of all transfers that have been
+            created and accepted by this account.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/EntityTransfer'
+                  page:
+                    $ref: '#/components/schemas/PaginationEnvelope/properties/page'
+                  pages:
+                    $ref: '#/components/schemas/PaginationEnvelope/properties/pages'
+                  results:
+                    $ref: '#/components/schemas/PaginationEnvelope/properties/results'
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Authorization: Bearer $TOKEN" \
+            https://api.linode.com/v4/account/entity-transfers
+      - lang: CLI
+        source: >
+          linode-cli account entity-transfers \
+            list
+    post:
+      x-linode-grant: unrestricted only
+      tags:
+      - Account
+      summary: Entity Transfer Create
+      description: >
+        Creates a transfer request for the specified entities. A request can contain any of the specified entity types
+        and any number of each entity type. When created successfully, a confirmation email is sent to the account
+        that created this transfer with instructions on completing the transfer. When completed, a transfer will move
+        the requested entities from the sending account to the receiving account.
+
+
+        This command can only be accessed by the unrestricted users of an account.
+
+
+        There are several conditions that must be met in order to successfully create a transfer request:
+
+          1. The account creating the transfer must not have a past due balance.
+
+          1. The entity must be owned by the account that is creating the transfer and must not be assigned to another
+          Entity Transfer or have any active Terms of Service violations.
+
+          1. Linodes must not be assigned to a NodeBalancer, Firewall, or Managed Service, have any attached Block
```

Comment:
@bbiggerr I think this may be better served as a bullet sub-list of checks, to make it easier for the user to go through.